WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -466,8 +464,8 @@ - (void)configureBlogCell:(WPBlogTableViewCell *)cell atIndexPath:(NSIndexPath *
     }
     cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
     cell.selectionStyle = self.tableView.isEditing ? UITableViewCellSelectionStyleNone : UITableViewCellSelectionStyleBlue;
-    cell.imageView.layer.borderColor = [UIColor whiteColor].CGColor;
-    cell.imageView.layer.borderWidth = 1.5;
+    cell.imageView.layer.borderColor = self.tableView.separatorColor.CGColor;
```

Comment:
Hmm it is, although the rows do have a selection / mouseover state with a coloured background. @mattmiklic could we get your input here? This is for the borders of the site images in the blog list.